test/python/test_notebooks.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2018, IBM.
+#
+# This source code is licensed under the Apache License, Version 2.0 found in
+# the LICENSE.txt file in the root directory of this source tree.
+
+# pylint: disable=redefined-builtin
+
+"""Tests for the wrapper functionality."""
+
+import os
+import unittest
+import subprocess
+import tempfile
+from .common import QiskitTestCase
+
+
+def _exec_notebook(path):
+    with tempfile.NamedTemporaryFile(suffix=".ipynb") as fout:
+        args = ["jupyter", "nbconvert", "--to", "notebook", "--execute",
+                "--ExecutePreprocessor.timeout=1000",
+                "--output", fout.name, path]
+    subprocess.check_call(args)
+
+
+class TestJupyter(QiskitTestCase):
+    """Notebooks test case."""
+    def setUp(self):
+        self.path = os.path.dirname(os.path.realpath(__file__))
+
+    def test_jupyter(self):
+        "Test Jupyter functionality"
+        success = True
+        try:
+            _exec_notebook(self.path+'/notebooks/test_jupyter.ipynb')
+        except Exception as excep:  # pylint: disable=w0703
+            success = False
+            print(excep)
+        self.assertTrue(success)
```

Comment:
Can we add an error message here? Right now it will fail with an error message `False is not True` which will be a bit confusing when trying to debug a failure. Yes it prints out the exceptions encountered during the run, but it's not always clear those map together (especially if running tests in parallel).

Actually thinking about it a bit more now why do we need to wrap `_exec_notebook()` in a try except? If that raises an exception then the tests failed. I agree that tests should have an assert though, maybe change the `subprocess.check_call()` at L24 to just `subprocess.call()` and assert the return code is 0.

Reply:
I can have it just throw the errors and not catch it in an assert.  I was not sure if these would get handled nicely by the unittests.

Reply:
Unittest will just print the traceback and error message from an unhandled exception and treat that as the failure.